src/olympia/constants/blocklist.py

```diff
@@ -1,8 +1,15 @@
 # How many guids should there be in the stashes before we make a new base.
+from olympia.blocklist.models import BlockType
+
+
 BASE_REPLACE_THRESHOLD = 5_000
 
 # Config keys used to track recent mlbf ids
 MLBF_TIME_CONFIG_KEY = 'blocklist_mlbf_generation_time'
-MLBF_BASE_ID_CONFIG_KEY = 'blocklist_mlbf_base_id'
+
+
+def MLBF_BASE_ID_CONFIG_KEY(block_type: BlockType):
+    return f'blocklist_mlbf_base_id_{block_type.name.lower()}'
```

Comment:
That's a good call out. We can manually set those values in our environments during the push and should just add that as a todo. Worst case scenario it would rcreate the base filters for both on the first run, then set the correct values.